tools/login/actions/login.php

```diff
@@ -125,8 +147,13 @@
         if ($existingUser["password"] === md5($_POST["password"])) {
             $this->SetUser($existingUser, $_POST["remember"]);
 
+            // si l'on veut utiliser l'url spécifié par $loggedinurl
+            if (isset($loggedinurl))
```

Comment:
```suggestion
            if (!empty($loggedinurl)){
              if ($this->IsWikiName($loggedinurl, WN_CAMEL_CASE_EVOLVED))
```
pour être plus stable

Reply:
isset vérifie si elle est définie, !empty vérifie qu'elle n'est pas nulle. empty interprete la chaine pour savoir si elle vaut vrai ou faux. Ce qui m'intéresse c'est essentiellement de savoir si elle est définie.